Sources/MessagePack/Decoder/SingleValueDecodingContainer.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+#if os(Linux)
+let NSEC_PER_SEC: UInt64 = 1000000000
```

Comment:
I wouldn't rely on this const as it's not available in Linux Foundation (for reason unknown).

Second thought: you should add another Travis check, on Linux environment.